<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,251,247)</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>10</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: rgb(155, 17, 30);
	color: #FFF;
}</string>
      </property>
      <property name="text">
       <string>Bill Splitter</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,251,247)</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page_1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="NumPeopleLabel">
            <property name="text">
             <string>Number in Group</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NumPeopleLineEdit">
            <property name="placeholderText">
             <string>Enter a number of people in a group</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="AddPeopleLabel">
            <property name="text">
             <string>Add People</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="nameLineEdit">
              <property name="placeholderText">
               <string>Enter a Name </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addNameButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>5</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    border-radius: 3px;
    background: rgb(0, 85,  255);
    color: white;
}

QPushButton:hover {
    background: rgba(0, 85, 255, 220);
}

QPushButton:pressed {
    background: rgba(0, 85, 255, 220);
}</string>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>100</height>
             </size>
            </property>
           </spacer>
          </item>
          <item alignment="">
           <widget class="QPushButton" name="movePageTwoButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>10</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border-radius: 3px;
    background: rgb(0, 85,  255);
    color: white;
}

QPushButton:hover {
    background: rgba(0, 85, 255, 220);
}

QPushButton:pressed {
    background: rgba(0, 85, 255, 220);
}</string>
            </property>
            <property name="text">
             <string>Submit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0">
           <property name="verticalSpacing">
            <number>10</number>
           </property>
           <item row="3" column="1">
            <widget class="QPushButton" name="debtShareButton">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    border-radius: 3px;
    background: rgb(0, 85,  255);
    color: white;
}

QPushButton:hover {
    background: rgba(0, 85, 255, 220);
}

QPushButton:pressed {
    background: rgba(0, 85, 255, 220);
}</string>
             </property>
             <property name="text">
              <string>Debt Share</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="TransHistoryButton">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    border-radius: 3px;
    background: rgb(0, 85,  255);
    color: white;
}

QPushButton:hover {
    background: rgba(0, 85, 255, 220);
}

QPushButton:pressed {
    background: rgba(0, 85, 255, 220);
}</string>
             </property>
             <property name="text">
              <string>Transaction History</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Maximum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>200</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="newTransactionButton">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    border-radius: 3px;
    background: rgb(0, 85,  255);
    color: white;
}

QPushButton:hover {
    background: rgba(0, 85, 255, 220);
}

QPushButton:pressed {
    background: rgba(0, 85, 255, 220);
}</string>
             </property>
             <property name="text">
              <string>+ New Transaction</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Maximum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>200</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QFrame" name="frame_2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>40</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="rightMargin">
            <number>43</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="goPage1Button">
               <property name="text">
                <string>Back</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="1">
         <widget class="QFrame" name="frame_3">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(75, 255, 132)</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <widget class="QPushButton" name="calculateButton">
             <property name="text">
              <string>Calculate Bill</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="billTotalLabel">
               <property name="text">
                <string>Bill Total</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="billTotalLineEdit"/>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>200</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0" rowspan="4">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="payerLabel">
            <property name="text">
             <string>Payer</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="payerLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="subTotalLabel">
            <property name="text">
             <string>Subtotal</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="subtotalLineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="gratuityLabel">
            <property name="text">
             <string>Gratuity</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLineEdit" name="gratuityLineEdit"/>
          </item>
          <item row="6" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="tipLabel">
              <property name="text">
               <string>Tip: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="tipLineEdit"/>
            </item>
           </layout>
          </item>
          <item row="7" column="0">
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QPushButton" name="fifteenTipButton">
               <property name="text">
                <string>15%</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="eighteenTipButton">
               <property name="text">
                <string>18%</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="twentyTipButton">
               <property name="text">
                <string>20%</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="twentyFiveTipButton">
               <property name="text">
                <string>25%</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="1">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="splitEvenlyButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Split/Add Fees Evenly</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="splitIndButton">
            <property name="text">
             <string>Split Fees Individually</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>80</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Total</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>70</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_9">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>220</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Name of debtor</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_10">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>220</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Share Amount</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_14">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>200</y>
          <width>141</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Fee</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>10</y>
          <width>331</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Split Bills Individually</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
